sdk/messaging/azservicebus/admin/admin_client_queue.go

```diff
@@ -329,8 +298,7 @@ type ListQueuesRuntimePropertiesOptions struct {
 
 // ListQueuesRuntimePropertiesResponse contains the page response for QueueRuntimePropertiesPager.PageResponse
 type ListQueuesRuntimePropertiesResponse struct {
-	Items       []*QueueRuntimePropertiesItem
-	RawResponse *http.Response
+	Items []QueueRuntimePropertiesItem
```

Comment:
`QueueRuntimeProperties []QueueRuntimePropertiesItem`

Reply:
Done. Although I did think QueueRuntimeProperties is kind of a funny name but it seemed okay in context.